tests/boto3_mocks.py

```diff
@@ -18,6 +18,7 @@ def func_wrapper():
         real_adapter_send = HTTPAdapter.send
         func()
         HTTPAdapter.send = real_adapter_send
+    func_wrapper.__doc__ = func.__doc__  # Replace docstring of inner function with the original.
```

Comment:
love this - just saw this somewhere else recently and is super nice to avoid duplicative doc strings 💯